<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JsonEditor</class>
 <widget class="QMainWindow" name="JsonEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="4,3,0">
    <item row="0" column="0" rowspan="5">
     <widget class="QPlainTextEdit" name="teScript">
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
      <property name="placeholderText">
       <string>JSON script</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="5">
     <widget class="QTreeView" name="treeView"/>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lKey">
        <property name="text">
         <string>Key</string>
        </property>
        <property name="buddy">
         <cstring>leKey</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leKey"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lType">
        <property name="text">
         <string>Type</string>
        </property>
        <property name="buddy">
         <cstring>cbType</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbType"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lValue">
        <property name="text">
         <string>Value</string>
        </property>
        <property name="buddy">
         <cstring>leValue</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leValue"/>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btnEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>364</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btnAddChild">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add child</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="btnAddSibling">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add sibling</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="actClose"/>
  </widget>
  <action name="actOpen">
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>open JSON file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>save JSON file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actClose">
   <property name="text">
    <string>close</string>
   </property>
   <property name="toolTip">
    <string>close JSON file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
